src/sql/src/normalize.rs

```diff
@@ -134,6 +134,17 @@ pub fn option_objects(options: &[SqlOption<Aug>]) -> BTreeMap<String, SqlOption<
         .collect()
 }
 
+pub fn with_options(options: &[WithOption]) -> BTreeMap<String, Value> {
+    options
+        .iter()
+        .filter_map(|o| match &o.value {
+            Some(WithOptionValue::Value(v)) => Some((o.key.to_string(), v.clone())),
+            Some(_) => None,
+            None => None,
```

Comment:
FWIW, I believe that Sean's other point was that this is dropping options on the floor, which is probably not the desired semantics.

Reply:
Yeah, I am not sure how I failed to realize there were only two variants and the other was equally easy to handle and instead wrote that. But either way that code is now gone. :)